comparecompanies.py

```diff
@@ -1,9 +1,11 @@
 import sys
 import numpy as np
-
+import parser as pa
```

Comment:
I rarely see modules renamed. Numpy as BeautifulSoup are the only 2 examples I see consistently. I generally don't alias my own modules in order to keep them readable ("pa" is a not-well-known abbreviation).